.github/workflows/load-generate.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: load-generate
+
+on:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.11'
+
+      - run: |
```

Comment:
@uchitsa I suspect, it's redundant, see: https://github.com/actions/setup-python?tab=readme-ov-file#caching-packages-dependencies